solr/core/src/test/org/apache/solr/logging/TestLogWatcher.java

```diff
@@ -104,4 +121,56 @@ public void testLog4jWatcher() throws InterruptedException {
       oldMessages.add(msg);
     }
   }
+
+  /**
+   * Here we validate that serialization works as expected for several different methods. Ideally we
+   * would use actual serialization from Jersey/Jackson, since this is what really happens in V2
+   * APIs. But this is simpler, and should give us roughly equivalent assurances.
+   */
```

Comment:
Does this seem reasonable? Am I missing a straightforward and/or better way to directly validate Jersey/Jackson serialization? cc @gerlowskija

Reply:
Apologies for not seeing this sooner @magibney 

I like what you have here, for a "unit" level test.

The alternative would be to test this higher up - say, by using SolrJettyTestCase and using SolrJ to hit the v2 logging API.  That'd test the serialization by virtue of testing the whole endpoint.

But I like your more-specific approach here better 👍

Reply:
No problem, and thanks for the review/suggestion! I'll probably go ahead and commit as-is; have to make sure this gets in 9.6.